tools/isobuild/bundler.js

```diff
@@ -2517,7 +2533,10 @@ class JsImage {
           from: nmd.sourcePath,
           to: nmd.preferredBundlePath,
           npmDiscards: nmd.npmDiscards,
-          symlink: includeNodeModules === 'symlink'
+          symlink: includeNodeModules === 'symlink' || (
```

Comment:
This is something I was not completely sure about. Not symlinking here actually breaks things in most situations if `node_modules` contain packages with deep symlink structure. On the other hand, symlinking does not make `bundle` self-contained. I think that at the end of the day, how exactly `node_modules` are included in the final `bundle` mostly depends on the monorepo manager, so perhaps we should add a third option to completely skip copying `node_modules` and leave this task to other tool.